DashAI/front/src/styles/globalComponents.js

```diff
@@ -1,22 +1,23 @@
 import styled from "styled-components";
-import { Button, Card, FloatingLabel, Form } from "react-bootstrap";
+// import { Button, Card, FloatingLabel, Form } from "react-bootstrap";
+// import { Card, FloatingLabel, Form } from "react-bootstrap";
 
-export const StyledButton = styled(Button)`
-  background-color: ${(props) => props.theme.button.background};
-  color: ${(props) => props.theme.button.text};
-  height: 2.2rem;
-  border-color: ${(props) => props.theme.button.border};
-  &:hover {
-    background-color: ${(props) => props.theme.button.backgroundHover};
-    color: ${(props) => props.theme.button.textHover};
-    border-color: ${(props) => props.theme.button.borderHover};
-  }
-  &:focus {
-    background-color: ${(props) => props.theme.button.backgroundFocus};
-    color: ${(props) => props.theme.button.textFocus};
-    box-shadow: none;
-  }
-`;
+// export const StyledButton = styled(Button)`
+//   background-color: ${(props) => props.theme.button.background};
+//   color: ${(props) => props.theme.button.text};
+//   height: 2.2rem;
+//   border-color: ${(props) => props.theme.button.border};
+//   &:hover {
+//     background-color: ${(props) => props.theme.button.backgroundHover};
+//     color: ${(props) => props.theme.button.textHover};
+//     border-color: ${(props) => props.theme.button.borderHover};
+//   }
+//   &:focus {
+//     background-color: ${(props) => props.theme.button.backgroundFocus};
+//     color: ${(props) => props.theme.button.textFocus};
+//     box-shadow: none;
+//   }
+// `;
```

Comment:
Delete commented code (remember that we can access to this code through a commit checkout).

```suggestion
```